### DISCUSSION TITLE
Streamlink on termux + vlc

### DISCUSSION THREAD

**Discussion by urgoz0** [streamlink/streamlink#5986] [Q&A] [answered]
### Description

I'm trying to make streamlink work on termux using android version of vlc player. I found this awesome guide from TwilightZer0, and installed termux and streamlink on my phone:

https://www.reddit.com/r/Streamlink/comments/cfno18/guide_streamlink_on_android_termux_vlc/

However, i dont have any luck to make it work with android vlc.
I tried to edit streamlink config file to set the player with termux's activity manager:

```
player=am start -n org.videolan.vlc/org.videolan.vlc.gui.video.VideoPlayerActivity -a android.intent.action.VIEW -d
player-http
```
but when i do it, streamlink keep saying "Player executable not found"

The am command is working from console, i can open any video or audio stream using the termux command line.
The only way to make it work to move am's attributes to player-args line:

```
player=am
player-http
player-args=start -n org.videolan.vlc/.StartActivity -a android.intent.action.VIEW -d
```
But when i do this, the stream is closing after a few seconds.

Any hint how to solve the problem?

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink/discussions/5913#discussioncomment-8972638
